indico/modules/rb_new/client/js/common/bookings/BookingEdit.jsx

```diff
@@ -0,0 +1,622 @@
+                <Modal.Header styleName="booking-edit-modal-header">
+                    <Translate>Edit a booking</Translate>
+                    {actionButtons()}
+                </Modal.Header>
+                <Modal.Content>
+                    <Grid columns={2}>
+                        <Grid.Column>
+                            <RoomBasicDetails room={room} />
+                            {this.renderForm(fprops)}
+                        </Grid.Column>
+                        <Grid.Column stretched>
+                            {this.renderBookingCalendar()}
+                            {this.renderConflictsMessage()}
+                        </Grid.Column>
+                    </Grid>
+                </Modal.Content>
+                <Modal.Actions>
+                    <Button type="submit"
+                            form="booking-edit-form"
+                            disabled={submitBlocked || conflictingBooking}
+                            primary>
+                        <Translate>Save changes</Translate>
+                    </Button>
+                    <Button type="button" onClick={onClose}>
+                        <Translate>Close</Translate>
+                    </Button>
+                </Modal.Actions>
+            </Modal>
+        );
+    };
+
+    renderForm = (fprops) => {
+        const {form, handleSubmit} = fprops;
+        const {values: {dates, recurrence, timeSlot, usage, user}, submitting, submitSucceeded} = fprops;
+        const notifyRecurrenceTypeChange = _.partial(this.notifyRecurrenceTypeChange, fprops);
+        const {user: sessionUser, booking: {bookedForUser}} = this.props;
+        const bookedByCurrentUser = sessionUser.id === bookedForUser.id;
+
+        return (
+            <Form id="booking-edit-form" styleName="booking-edit-form" onSubmit={handleSubmit}>
+                <Segment>
+                    <Form.Group inline>
+                        <Field name="recurrence.type"
+                               component={ReduxRadioField}
+                               as={Radio}
+                               componentLabel={Translate.string('Single booking')}
+                               radioValue="single"
+                               disabled={submitting || submitSucceeded}
+                               onClick={() => notifyRecurrenceTypeChange('single')} />
+                        <Field name="recurrence.type"
+                               component={ReduxRadioField}
+                               as={Radio}
+                               componentLabel={Translate.string('Daily booking')}
+                               radioValue="daily"
+                               disabled={submitting || submitSucceeded}
+                               onClick={() => notifyRecurrenceTypeChange('daily')} />
+                        <Field name="recurrence.type"
+                               component={ReduxRadioField}
+                               as={Radio}
```

Comment:
What about moving making this a default in `ReduxRadioField` like I did in `ReduxDropdownField`?